### PR TITLE
Renamed email address configuration for outlook 2016 on mac

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d95dfaaa6f/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-17470da017/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-07fa85af08/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-efebd6105b/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-73727fd397/how-to/)

**Comment by catlook** [comment]
Kelly, please work your magic on this one!

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3982068085/how-to/)

**Comment by kmsholcomb** [comment]
Hi @catlook. My suggested edits are attached. Please let me know if you have any questions. Thanks!

[Configure a renamed email address on Outlook 2016 for Mac_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1293879/Configure.a.renamed.email.address.on.Outlook.2016.for.Mac_editsKH.docx)
[Configure a renamed email address on Outlook 2016 for Mac_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1293880/Configure.a.renamed.email.address.on.Outlook.2016.for.Mac_editsKH.pdf)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c96261d730/how-to/)

File: content/rackspace-email/renamed-email-address-configuration-for-outlook-2016-on-mac.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+permalink: renamed-email-address-configuration-for-outlook-2016-on-mac/
```
Comment (DonNateR) [inline content/rackspace-email/renamed-email-address-configuration-for-outlook-2016-on-mac.md]: File name and permalink should be changed to reflect the new title.

Comment (catlook) [reply content/rackspace-email/renamed-email-address-configuration-for-outlook-2016-on-mac.md]: doh!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e4c785781a/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7bfb25158a/how-to/).

**Pull request merged**